ipaserver/plugins/fido2config.py

```diff
@@ -0,0 +1,95 @@
+#
+# Copyright (C) 2022  FreeIPA Contributors see COPYING for license
+#
+
+import logging
+
+from ipalib import api
+from ipalib.parameters import StrEnum
+from ipalib.plugable import Registry
+from .baseldap import (
+    LDAPObject,
+    LDAPRetrieve,
+    LDAPUpdate)
+from ipalib import _
+
+
+logger = logging.getLogger(__name__)
+
+__doc__ = _("""
+FIDO2 configuration
+""") + _("""
+Manage FIDO2 configuration.
+""") + _("""
+IPA supports the use of FIDO2 devices for authentication. A FIDO2
+device has to be registered to SSSD and the resulting authentication mapping
+stored in the user entry.
+The FIDO2 authentication supports the following configuration option:
+require user verification. When set, the user must provides the FIDO2 device's
+PIN during authentication.
```

Comment:
The user verification can be done in several ways (i.e. PIN, fingerprint, external pad) so I would avoid mentioning the PIN or just using it as an example.

Reply:
Reworded.